scss/_custom-forms.scss

```diff
@@ -140,9 +140,9 @@
   background-size: $custom-select-bg-size;
   border: $custom-select-border-width solid $custom-select-border-color;
   @if $enable-rounded {
-    border-radius: $custom-select-border-radius;
+    @include border-radius($custom-select-border-radius);
```

Comment:
There is no need to use the mixin here, or replace the if statement with the mixin